modules/_labs/select/react/lib/SelectMenu.tsx

```diff
@@ -1,17 +1,45 @@
 import * as React from 'react';
-import {ErrorType, styled} from '@workday/canvas-kit-react-common';
+import {Rect} from '@popperjs/core';
```

Comment:
Should we export `Rect` from `@popperjs/core` in the Canvas Kit Popper (in addition to `Placement` and `PopperOptions`)? Currently, I'm using `Rect` to type the `reference` and `popper` arguments in the `offset` modifier.

Reply:
Feel free to do that

Reply:
Done.